File: JavaScript/7js 1/babySharkLyrics.js
/*
|--------------------------------------------------------------------------
| Baby shark lyrics generator
|--------------------------------------------------------------------------
|
*/

function babySharkLyrics() {
   let x = [
      "Baby shark",
      "Mommy shark",
      "Daddy shark",
      "Grandma shark",
      "Grandpa shark",
      "Let's go hunt",
   ];
   return x.map((i) => `${i},${" doo".repeat(6)}\n`.repeat(3) + `${i}!\n`).join("") + "Run away,…";
}


function babySharkLyrics() {
   return (
      [
         ...[`Baby`, `Mommy`, `Daddy`, `Grandma`, `Grandpa`].map((v) => v + ` shark`),
         `Let's go hunt`,
      ].map((v) => `${v},${` doo`.repeat(6)}\n`.repeat(3) + v + `!\n`).join`` + `Run away,…`
   );
}


function babySharkLyrics() {
   let d = " doo".repeat(6),
      a = ["Baby", "Mommy", "Daddy", "Grandma", "Grandpa"],
      r = "";
   for (let i in a) {
      r += (a[i] + " shark," + d + "\n").repeat(3);
      r += a[i] + " shark!\n";
   }
   return r + ("Let's go hunt," + d + "\n").repeat(3) + "Let's go hunt!\nRun away,…";
}


console.log(babySharkLyrics());

File: Python/7py 1/baby_shark_lyrics.py
"""
|--------------------------------------------------------------------------
| Baby shark lyrics generator
|--------------------------------------------------------------------------
|
"""

def baby_shark_lyrics():
   x = [
      "Baby shark",
      "Mommy shark",
      "Daddy shark",
      "Grandma shark",
      "Grandpa shark",
      "Let's go hunt",
   ]
   return ''.join([f"{i},{' doo'*6}\n"*3 + f"{i}!\n" for i in x]) + "Run away,…"


def baby_shark_lyrics():
   arr = ['Baby shark','Mommy shark','Daddy shark','Grandma shark','Grandpa shark',"Let's go hunt"]
   out = ''
   doo =' doo' * 6
   for i in arr:
      out += (i+','+doo+'\n')*3 + (i+'!\n')
   return out + 'Run away,…'


def baby_shark_lyrics():
   a = "{}," +  " doo" * 6 + '\n'
   b = [x + ' shark' for x in ['Baby', 'Mommy', 'Daddy', 'Grandma', 'Grandpa']] + ["Let's go hunt"]
   return ''.join([a.format(x)*3 + x + '!\n' for x in b]) + 'Run away,…'


print(baby_shark_lyrics())